source/docs/software/vision-processing/calibration.rst

```diff
@@ -1,119 +1,94 @@
+.. include:: <isogrk3.txt>
+.. include:: <isonum.txt>
+
 Calibration
 ===========
-While many of the numbers for the Vision Processing code can be determined theoretically, there are a few parameters that
-are typically best to measure empirically then enter back into the code (a process typically known as calibration). This
-article will show how to perform calibration for the Color (masking), and View Angle (distance) using the NI Vision
-Assistant. If you are using C++ or Java and have not yet installed the NI Vision Assistant, see the article Installing NI
-Vision Assistant.
+
+While many of the numbers for the Vision Processing code can be determined theoretically, there are a few parameters that are typically best to measure empirically then enter back into the code (a process typically known as calibration). This article will show how to perform calibration for the Color (masking), and View Angle (distance) using the NI Vision Assistant. If you are using C++ or Java and have not yet installed the NI Vision Assistant, see the article Installing NI Vision Assistant.
 
 Enable Snapshots
 ----------------
+
 .. figure:: images/calibration/enable-snapshots.png
```

Comment:
```suggestion
 .. image:: images/calibration/enable-snapshots.png
```

Reply:
Why the preference for image over figure?  Should we always be using image?

Reply:
I don't have anything against it, more so consistency with other areas of documentation. Should we be using figure?

Reply:
After looking more at it, I think we should use image unless we're going to put captions everywhere.  Of course, that brings up whether we should put captions everywhere, which maybe we should, but is a lot of work.  I'll just switch to image for now.